docs/en/integrations/data-ingestion/kafka/index.md

```diff
@@ -754,13 +754,26 @@ Navigate to “Connectors” -> “Add Connector” and use the following settin
   "value.converter.schemas.enable": "false"
 }
 ```
+#### Specify the connection endpoints
+You need to specify the allow-list of endpoints that the connector can access.
+You must use a fully-qualified domain name (FQDN) when adding the networking egress endpoint(s).
+Example: `u57swl97we.eu-west-1.aws.clickhouse.com:8443`
+
+:::note
+You must specify HTTP(S) port. The Connector doesn't support Native protocol yet.
+:::
+
+[Read the documentation.](https://docs.confluent.io/cloud/current/connectors/bring-your-connector/custom-connector-qs.html#cc-byoc-endpoints)
+
 You should be all set!
 
 ##### Known Limitations
+* Custom Connectors must use public internet endpoints. Static IP addresses aren't supported.
 * You can override some Custom Connector properties. See the fill [list in the official documentation.](https://docs.confluent.io/cloud/current/connectors/bring-your-connector/custom-connector-manage.html#override-configuration-properties)
 * Custom Connectors are available only in [some AWS regions](https://docs.confluent.io/cloud/current/connectors/bring-your-connector/custom-connector-fands.html#supported-aws-regions)
 * See the list of [Custom Connectors limitations in the official docs](https://docs.confluent.io/cloud/current/connectors/bring-your-connector/custom-connector-fands.html#limitations)
-
+#### Installing on AWS MSK
```

Comment:
@rfraposa why did you merge all the Kafka-related docs into a single page in this PR https://github.com/ClickHouse/clickhouse-docs/pull/850? Do you know if it is done accidentally?

Reply:
No - it was deliberate. At the time, Kafka (and quite a few other topics) were spread across multiple pages. It was an attempt to make the docs easier to navigate and find what you need - by reducing the number of docs pages and consolidating those topics to a single page.

This was one of several PRs that condensed topics to a single page.

Reply:
@rfraposa  I use the page regularly and find it unfriendly: it's almost impossible to search for something on the page due to its size. I want to revert these changes to the structure we introduced in https://github.com/ClickHouse/clickhouse-docs/pull/740.

Reply:
Probably a conversation to have with @justindeguzman instead of me.

Reply:
@mshustov it'd be good to sync with you and @Ryado on this for the Kafka docs rewrites